build-support/bin/check_pants_pex_abi.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python2.7
```

Comment:
This is so much better than the bash script. Thanks for the advice to do this and the bash tips as well! I incorporated a lot of your ideas, e.g. only grabbing the file we need from unzip rather than unzipping the whole thing.